docs/all_rectors_overview.md

```diff
@@ -100,7 +100,30 @@ Refactor file ext_emconf.php
 
 ### GeneralUtilityMakeInstanceToConstructorPropertyRector
 
-Move GeneralUtility::makeInstance calls to constructor injection
+Move GeneralUtility::makeInstance calls to constructor injection.
+
+> [!IMPORTANT]
+> If you use this rule, please add the following snippet to your composer.json:
+
+```json
+{
+    "extra": {
+        "patches": {
+            "rector/rector": [
+                "https://raw.githubusercontent.com/sabbelasichon/typo3-rector/refs/heads/2.x/patches/rector-rector-rules-typedeclaration-nodeanalyzer-autowiredclassmethodorpropertyanalyzer-php.patch",
+                "https://raw.githubusercontent.com/sabbelasichon/typo3-rector/refs/heads/2.x/patches/rector-rector-src-nodemanipulator-classdependencymanipulator-php.patch",
+                "https://raw.githubusercontent.com/sabbelasichon/typo3-rector/refs/heads/2.x/patches/rector-rector-src-phpparser-node-nodefactory-php.patch"
+            ]
+        }
+    }
+}
+```
+
+and execute:
+
+```bash
+composer config --no-plugins allow-plugins.cweagans/composer-patches true
+```
```

Comment:
This can be dropped since `cweagans/composer-patches` must be installed anyways, so the plugin will be allowed.